src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
+    }
```

Comment:
The 'ColorSupplier' class should not create a new instance of 'Random' every time the 'getRandomColor()' method is called. Instead, you should create a single instance of 'Random' as a class member (private field) and reuse it. This is more efficient as it avoids unnecessary object creation.